choreographer/src/main/java/eu/arrowhead/core/choreographer/database/service/ChoreographerSessionDBService.java

```diff
@@ -368,7 +434,7 @@ public ChoreographerSessionStep changeSessionStepExecutor(final long sessionStep
 			sessionStep.setExecutor(executorOpt.get());
 			sessionStep = sessionStepRepository.saveAndFlush(sessionStep);
 			
-			worklog(sessionStep.getSession().getPlan().getName(), sessionStep.getStep().getAction().getName(), sessionStep.getStep().getName(), sessionStep.getStep().getId(),
+			worklog(sessionStep.getSession().getPlan().getName(), sessionStep.getStep().getAction().getName(), sessionStep.getStep().getName(), sessionStep.getStep().getId(), sessionStep.getSession().getExecutionNumber(),
```

Comment:
we should create a temp variable from sessionStep.getStep() and sessionStep.getSession() to made this line shorter